test/features/helpers/i18n_helper.rb

```diff
@@ -0,0 +1,12 @@
+class I18nHelper
+  def load_locale(locale)
+    tag = locale ? locale : 'en'
+    lines = File.readlines(File.expand_path("../locales/#{tag}.js"))
+    lines[0] = "{\n"
+    @translations = eval(lines.join)
```

Comment:
I think we should be able to avoid `eval` here. Can't we just use a JSON reader?

Reply:
Yes, we can.

Reply:
yes please, eval can leave us open to code injection.

Reply:
Yes, good point guys. Done